pil-analyzer/tests/parse_display.rs

```diff
@@ -21,7 +21,7 @@ namespace std::convert(65536);
 namespace T(65536);
     col fixed first_step = [1] + [0]*;
     col fixed line(i) { i };
-    let ops: int -> bool = (|i| ((i < 7) && (6 >= -i)));
+    let ops: int -> bool = (|i| (i < 7) && (6 >= -i));
```

Comment:
```suggestion
    let ops: int -> bool = |i| i < 7 && 6 >= -i;
```

Reply:
Greater and greater than equal have RequireParentheses associativity

Reply:
Maybe then this is one example of what I was talking about: You only need parentheses with "RequireParentheses" when the other expression has the same precedence.

Reply:
Fixed.